README.md

```diff
@@ -71,15 +71,21 @@ use 'f4z3r/gruvbox-material.nvim'
 
 
 ```lua
-
 {
-  'f4z3r/gruvbox-material.nvim',
-  name = 'gruvbox-material',
-  lazy = false,
-  priority = 1000,
-  config = function()
-    -- configs go here
-  end,
+    'f4z3r/gruvbox-material.nvim',
+	name = 'gruvbox-material',
+	lazy = false,
+	priority = 1000,
+	opts = {
+        italics = true,                       		-- enable italics in general
```

Comment:
I don't think it is necessary to duplicate the entire configuration here. Maybe just add a comment that refers to the section below?

Also, I have the lua code to use 2 spaces for indentation (see `stylua.toml`). Can you also use this here please?